detect_regressions.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+import pathlib
+
+import pandas
+import sqlalchemy
+
+
+def detect_regressions(database_file):
+
+    engine = sqlalchemy.create_engine(f"sqlite:///{database_file}")
+
+    df = pandas.read_sql("select * from test_run where platform = 'linux'", engine)
```

Comment:
Since you also filter by `"passed"` below, we might as well do it here:
```suggestion
    df = pandas.read_sql("select * from test_run where platform = 'linux' and call_outcome = 'passed'", engine)
```